src/theme/widget.rs

```diff
@@ -59,19 +106,11 @@ where
     (
         Name::new("Button"),
         Node::default(),
-        Children::spawn(SpawnWith(|parent: &mut ChildSpawner| {
+        Children::spawn(SpawnWith(move |parent: &mut ChildSpawner| {
```

Comment:
```suggestion
        Children::spawn(SpawnWith(|parent: &mut ChildSpawner| {
```
Is the `move` still needed?